drivers/hmis/app/graphql/mutations/update_inventory.rb

```diff
@@ -0,0 +1,18 @@
+module Mutations
+  class UpdateInventory < BaseMutation
+    argument :id, ID, required: true
+    argument :input, Types::HmisSchema::InventoryInput, required: true
+
+    field :inventory, Types::HmisSchema::Inventory, null: true
+    field :errors, [Types::HmisSchema::ValidationError], null: false
+
+    def resolve(id:, input:)
+      record = Hmis::Hud::Inventory.viewable_by(current_user).find_by(id: id)
+      default_update_record(
+        record: record,
+        field_name: :inventory,
+        input: input,
+      )
+    end
```

Comment:
not sure it's worth it, but this could be cleaned up.  The resolve method could be moved to a concern and methods for `hud_class` and `field_name` could be added here.  Again, probably not worth it since this is very clean and simple, but if we end up needing to do more in these classes, then maybe.